vault/data_source_address_test.go

```diff
@@ -0,0 +1,28 @@
+package vault
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccDataSourceAddress(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testutil.TestAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceAddressBasic_config,
+				Check:  resource.TestCheckResourceAttrSet("data.vault_address.test", "address"),
```

Comment:
Not sure if there is a requirement for the address to begin with `http(s)://`, so maybe testing of the existence is enough?